interface/globals.php

```diff
@@ -610,4 +610,58 @@ function strterm($string, $length)
 }
 
 // turn off PHP compatibility warnings
+
 ini_set("session.bug_compat_warn", "off");
+
+
+/* Pages with "myadmin" in the URL don't need register_globals. */
+$fake_register_globals =
+	$fake_register_globals && (strpos($_SERVER['REQUEST_URI'],"myadmin") === FALSE);
+
+
+// Emulates register_globals = On.  Moved to the bottom of globals.php to prevent
+// overrides of any variables used during global setup.
+// EXTR_SKIP flag set to prevent overriding any variables defined earlier
+if ($fake_register_globals) {
+  extract($_GET,EXTR_SKIP);
+  extract($_POST,EXTR_SKIP);
+}
```

Comment:
then would remove this since codebase no longer uses these flags
(note this was removed because it was a security vulnerability.
doing this did unmask a couple bugs, but have been addressed.)